.gitignore

```diff
@@ -100,3 +100,6 @@ ENV/
 
 # Mac file caches
 .DS_Store
+
+# PyCharm project settings
+.idea/
```

Comment:
I think we probably don't need to commit this.

Reply:
I noticed that there were other varieties of editor project settings in the gitignore so I committed mine. Should the others go as well?